projects/packages/masterbar/src/class-main.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Main class for the Masterbar package.
+ *
+ * @package automattic/jetpack-masterbar
```

Comment:
The annotation for package here (`automattic/jetpack-masterbar`) seems to be different from most of the files (`jetpack-masterbar`)

Reply:
Fixed all package annotations with https://github.com/Automattic/jetpack/commit/d117431f364b2fb5b3df13cb65e859fce66b4981

The right one is indeed `@package automattic/jetpack-masterbar`